src/ServicePulse.Host/angular/app/css/particular.css

```diff
@@ -2608,14 +2608,14 @@ span.fa-exclamation-triangle.danger {
 }
 
 #toast-container > .toast-error {
-    background-image: url('../a/img/toast-danger.svg') !important;
+    background-image: url('../img/toast-danger.svg') !important;
     background-position-y: 30px;
     background-size: 28px 28px;
 }
 
 #toast-container > .toast-warning {
     background-color: #F3BC52;
-    background-image: url('../a/img/toast-warning.svg') !important;
+    background-image: url('../img/toast-warning.svg') !important;
```

Comment:
These changes are on the angular side. Were the toasts images  not showing as expected even on the angular pages?

Reply:
This actually fixes the angular toast warning display by adding the warning icon  
<img width="204" alt="image" src="https://github.com/Particular/ServicePulse/assets/87037242/36d3cf5e-3f2a-4558-8a51-2ffca6c98ee2">

This image is from the feature branch without the update to the path
<img width="203" alt="image" src="https://github.com/Particular/ServicePulse/assets/87037242/dab06305-6284-43af-8890-431ccfd89e7c">